mkdocs.yml

```diff
@@ -73,7 +73,61 @@ plugins:
   - redirects:
       redirect_maps:
         'tutorial/index.md': 'guides/pet-shop/index.md'
-        'tutorial/pet-shop/index.md': 'guides/pet-shop/index.md'          
+        'tutorial/pet-shop/index.md': 'guides/pet-shop/index.md'
         'tutorials/index.md': 'guides/pet-shop/index.md'
         'tutorials/pet-shop/index.md': 'guides/pet-shop/index.md'
         'docs/ganache/overview/index.md': 'docs/ganache/index.md'
+#       Truffle docs restructure
```

Comment:
If we're already redirecting most of the links in this PR, this might be a good time to review and update the file and directory names for consistency and simplicity (to avoid having to add *double* the amount of redirects down the line).

Some nitpicks on naming:
- Since our headings should use simple present tense verbs (e.g., "Use" instead of "Using"), the file names should match.
- I prefer "how-to" over "howto" for the directory name to be consistent with using dashes for spaces.
- Some page titles are changed in the restructure, so the file name should change to match (e.g., "Choosing an Ethereum client" to "Ethereum client types").

Reply:
I purposely avoided doing the renaming etc in this PR. But you're right, it makes more sense to do it now. I didn't think about having to add extra redirects down the line if I hold off.